app/src/main/java/org/oppia/app/topic/TopicFragmentPresenter.kt

```diff
@@ -87,4 +93,49 @@ class TopicFragmentPresenter @Inject constructor(
   private fun getTopicViewModel(): TopicViewModel {
     return viewModelProvider.getForFragment(fragment, TopicViewModel::class.java)
   }
+
+  private fun logTopicEvents(tab: TopicTab){
+    when(tab){
```

Comment:
space after `when` and before `{`, If you try doing `reformat code` from the menu toolbar such issues resolves automatically.

Reply:
Done.